CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - The Conjur version is now printed on server startup, after running `conjurctl server`
   ([cyberark/conjur#1590](https://github.com/cyberark/conjur/pull/1590))
 
+### Security
+- Upgrade Rails to `v5.2.4.3` to resolve [CVE-2020-8164](https://groups.google.com/forum/#!topic/rubyonrails-security/f6ioe4sdpbY).
```

Comment:
@andytinkham gave us guidance that we don't need to document dependency upgrades in the security section.  We should be able to remove this line.

Reply:
To be clear, I said it was TBD and I was leaning towards not including them, so don't take it as gospel truth yet. Naama is raising points in another thread that make this still very much TBD. I'm OK either removing this line right now or just leaving off the "to resolve" bit.

Reply:
The line is removed